### PR TITLE
feat: plugin Submission: CommandBar + Netlify

### PR DESCRIPTION
**Pull request by rfreling**
For submission/review! 🔍  ⚡

Thanks for contributing the Netlify plugins directory!

**Are you adding a plugin or updating one?**

- [x] Adding a plugin
- [ ] Updating a plugin

**Have you completed the following?**

- [x] Read and followed the [plugin author guidelines](https://github.com/netlify/plugins/blob/main/docs/guidelines.md).
- [x] Included all [required fields](https://github.com/netlify/plugins/blob/main/docs/CONTRIBUTING.md#required-fields) in your entry.
- [x] Tested the plugin [locally](https://docs.netlify.com/cli/get-started/#run-builds-locally) and [on Netlify](https://docs.netlify.com/configure-builds/build-plugins/#install-a-plugin), using the plugin version stated in your entry.

**Test plan**
Please add a link to a successful public deploy log using the stated version of the plugin. Include any other context reviewers might need for testing.

### CODE REVIEW

File: site/plugins.json [hunk 1]
```diff
@@ -609,5 +609,13 @@
...
-  }
+  },
+  {
+    "author": "CommandBar",
+    "description": "CommandBar plugin for sites deployed to Netlify",
+    "name": "CommandBar",
+    "package": "@commandbar/netlify-plugin-commandbar",
+    "repo": "https://github.com/tryfoobar/netlify-plugin-commandbar",
+    "version": "0.0.2"
+  },
```
Comment (ehmicky) [inline site/plugins.json:620]:
```suggestion
  }
```

**Comment by ehmicky** [comment]
Hi @rfreling,

Thanks a lot for this Build plugin, this is awesome!

I have added a couple of comments as [repository issues](https://github.com/tryfoobar/netlify-plugin-commandbar/issues/created_by/ehmicky), please let me know what you think.

Otherwise, it looks great! :tada: 

Small note: the commit message must be prefixed with `feat:` to follow Conventional commit guidelines.

File: site/plugins.json (hunk 1 above)
Comment (rfreling) [reply site/plugins.json]: 👍🏻  Fixed here! https://github.com/netlify/plugins/pull/631/commits/2b9685f565420dc09f2ab53394ced7dfcd2cb2c0

**Comment by rfreling** [comment]
Thanks a bunch @ehmicky ! And sorry for the slow reply.

The issues you suggested made a lot of sense, and I think we addressed all of them [[LINK](https://github.com/tryfoobar/netlify-plugin-commandbar/issues)]

Let us know if there's anything else that we should work on / update!

**Comment by ehmicky** [comment]
Nice, this is good to go! :tada: 
One last thing: you need to publish a new version on npm and update the version in this PR. Thanks!

**Review by ehmicky — APPROVED** [review]
:tada:

**Pull request merged**